package.json

```diff
@@ -45,33 +46,32 @@
     "font-awesome": "4.7.0",
     "jquery": "^3.6.0",
     "jqwidgets-ng": "^14.1.17",
-    "ngx-cookie-service": "^13.1.2",
-    "ngx-text-diff": "^0.6.0",
```

Comment:
Did you forget to add ngx-text-diff back?
And if so, test it so it works properly as it did before but with angular 16